spec/models/spree/printables/invoice/base_view_spec.rb

```diff
@@ -89,5 +92,23 @@
         base_view.number
       end
     end
+
+    context 'when using a NumberFormatter' do
+      before do
+        class Spree::PrintInvoice::NumberFormatter
+          def initialize(number)
+            @number = number
+          end
+
+          def to_s
+            "MY-NICE-INVOICE-#{@number}"
+          end
+        end
+      end
+
+      it 'returns a nicely formatted number' do
+        expect(base_view.number).to eq("MY-NICE-INVOICE-77")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.